### PR TITLE
Update go mod ref

### PR DESCRIPTION
**Pull request by jbutlerdev**
Update `hashicorp/vault` references to point to `lf-edge/openbao`

This will currently fail to build as the `github.com/lf-edge/openbao/vault/hcp_link/proto` needs to exist with the updated `go.mod` file before the `go.sum` file can be properly updated

### CODE REVIEW

**Comment by jbutlerdev** [comment]
#1

**Comment by Scorpil** [comment]
Hey @jbutlerdev, hcp-link is [still MPL 2.0](https://github.com/hashicorp/hcp-link/blob/main/LICENSE), couldn't we just use it directly?

**Comment by jbutlerdev** [comment]
> Hey @jbutlerdev, hcp-link is [still MPL 2.0](https://github.com/hashicorp/hcp-link/blob/main/LICENSE), couldn't we just use it directly?

That's correct, however since the module is hosted within the `vault` repository and therefor also within the `openbao` repository I thought it made sense to point to our forked version. Do you think we should still be pointing to `vault` for this?

**Comment by Scorpil** [comment]
Oh it's in the vault repo... Nevermind then, I assumed it's pointing to the separate repo, my bad. You're right, we should keep it as is.

**Comment by Scorpil** [comment]
Alright, to test the branch I've added 

```
replace github.com/lf-edge/openbao/vault/hcp_link/proto => ./vault/hcp_link/proto
```

to go.mod, this allows me to progress yet build fails on a compile step with

```
# github.com/lf-edge/openbao/helper/testhelpers/teststorage
helper/testhelpers/teststorage/teststorage.go:264:10: cannot use logicalKv.Factory (value of type func(ctx "context".Context, conf *"github.com/hashicorp/vault/sdk/logical".BackendConfig) ("github.com/hashicorp/vault/sdk/logical".Backend, error)) as "github.com/lf-edge/openbao/sdk/logical".Factory value in map literal
# github.com/lf-edge/openbao/helper/builtinplugins
helper/builtinplugins/registry.go:102:26: cannot use credAliCloud.Factory (value of type func(ctx "context".Context, conf *"github.com/hashicorp/vault/sdk/logical".BackendConfig) ("github.com/hashicorp/vault/sdk/logical".Backend, error)) as "github.com/lf-edge/openbao/sdk/logical".Factory value in struct literal

... and more of the same error...
```

Looks like `logicalKv` refers to `"github.com/hashicorp/vault-plugin-secrets-kv"` which of internally refers to

```
	github.com/hashicorp/vault/api v1.9.2
	github.com/hashicorp/vault/sdk v0.10.0
```

Causing the incompatibility. Any idea how to fix this short of forking `"github.com/hashicorp/vault-plugin-secrets-kv"`?

**Comment by jbutlerdev** [comment]
Good call on the replace statement. I'll give this a shot today and get back to you

**Comment by naphelps** [comment]
`.../vault/api` and `.../vault/sdk` both are still currently using MPL 2.0 license still. Short-term these references should not need to be changed.

Long-term if we needed to fork some of these plugins we would need to potentially work with OpenTofu to see if they use the same plugins or not, and what org the repo would fall under.

**Comment by naphelps** [comment]
Make sure all commits use `-s` or `--signoff` for sign off.

**Comment by Scorpil** [comment]
@naphelps that was my original thinking as well (see `hcp_link` discussion above). However, as @jbutlerdev pointed out those modules are already forked (they're in this repo), so pointing to vault repo would be a bit dirty: first commiter who needs to modify those crucial sub-modules will be forced to solve the issue...

Besides, in this particular case it's not quite possible because the dependency is transitive: `vault-plugin-secrets-kv` improts values from hashicorp's vault, so when our codebase tries to use those - type mismatch happens.

**Comment by naphelps** [comment]
Do we have a known good list of plugins that need to be additionally forked?

**Comment by cipherboy** [comment]
@naphelps said:
> Do we have a known good list of plugins that need to be additionally forked?

This should be correct:

```
$ grep 'hashicorp/vault-plugin-' ./go.mod`
	github.com/hashicorp/vault-plugin-auth-alicloud v0.15.0
	github.com/hashicorp/vault-plugin-auth-azure v0.16.0
	github.com/hashicorp/vault-plugin-auth-centrify v0.15.1
	github.com/hashicorp/vault-plugin-auth-cf v0.15.0
	github.com/hashicorp/vault-plugin-auth-gcp v0.16.0
	github.com/hashicorp/vault-plugin-auth-jwt v0.16.0
	github.com/hashicorp/vault-plugin-auth-kerberos v0.10.0
	github.com/hashicorp/vault-plugin-auth-kubernetes v0.16.0
	github.com/hashicorp/vault-plugin-auth-oci v0.14.0
	github.com/hashicorp/vault-plugin-database-couchbase v0.9.2
	github.com/hashicorp/vault-plugin-database-elasticsearch v0.13.2
	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.10.0
	github.com/hashicorp/vault-plugin-database-redis v0.2.1
	github.com/hashicorp/vault-plugin-database-redis-elasticache v0.2.1
	github.com/hashicorp/vault-plugin-database-snowflake v0.9.0
	github.com/hashicorp/vault-plugin-mock v0.16.1
	github.com/hashicorp/vault-plugin-secrets-ad v0.16.0
	github.com/hashicorp/vault-plugin-secrets-alicloud v0.15.0
	github.com/hashicorp/vault-plugin-secrets-azure v0.16.1
	github.com/hashicorp/vault-plugin-secrets-gcp v0.16.0
	github.com/hashicorp/vault-plugin-secrets-gcpkms v0.15.0
	github.com/hashicorp/vault-plugin-secrets-kubernetes v0.5.0
	github.com/hashicorp/vault-plugin-secrets-kv v0.15.0
	github.com/hashicorp/vault-plugin-secrets-mongodbatlas v0.10.2
	github.com/hashicorp/vault-plugin-secrets-openldap v0.11.1
	github.com/hashicorp/vault-plugin-secrets-terraform v0.7.1
```

Only one I'm not really sure about is `github.com/hashicorp/vault-plugin-mock`.

**Comment by Scorpil** [comment]
Should be easy enough to comment them out for initial fork though, then take them up one-by-one. `plugin-secrets-kv` is used in many tests so it's the only one that's a must, imo.

File: .github/workflows/test-go.yml [hunk 1]
```diff
@@ -391,7 +391,7 @@ jobs:
           # This jq query filters out successful tests, leaving only the failures.
           # Then, it formats the results into rows of a Markdown table.
           # An example row will resemble this:
-          # | github.com/hashicorp/vault/package | TestName | fips | 0 | 2 | [view results](github.com/link-to-logs) |
+          # | github.com/lf-edge/openbao/package | TestName | fips | 0 | 2 | [view results](github.com/link-to-logs) |
```
Comment (naphelps) [inline .github/workflows/test-go.yml:394]: openbao/openbao

File: .github/scripts/report_failed_builds.sh [hunk 2]
```diff
@@ -35,7 +35,7 @@ if [ ${#failed_jobs[@]} -eq 0 ]; then
 All builds succeeded! :white_check_mark:"
 else
   new_body="Build Results:
-Build failed for these jobs: ${failed_jobs[*]}. Please refer to this workflow to learn more: https://github.com/hashicorp/vault/actions/runs/$RUN_ID"
+Build failed for these jobs: ${failed_jobs[*]}. Please refer to this workflow to learn more: https://github.com/lf-edge/openbao/actions/runs/$RUN_ID"
```
Comment (naphelps) [inline .github/scripts/report_failed_builds.sh:38]: openbao/openbao

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@
 - Tutorials: [HashiCorp's Learn Platform](https://learn.hashicorp.com/vault)
 - Certification Exam: [Vault Associate](https://www.hashicorp.com/certification/#hashicorp-certified-vault-associate)
 
-<img width="300" alt="Vault Logo" src="https://github.com/hashicorp/vault/blob/f22d202cde2018f9455dec755118a9b84586e082/Vault_PrimaryLogo_Black.png">
+<img width="300" alt="Vault Logo" src="https://github.com/lf-edge/openbao/blob/f22d202cde2018f9455dec755118a9b84586e082/Vault_PrimaryLogo_Black.png">
```
Comment (naphelps) [inline README.md:16]: openbao/openbao

File: README.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ $ make test TEST=./vault
 ### Importing Vault
 
 This repository publishes two libraries that may be imported by other projects:
-`github.com/hashicorp/vault/api` and `github.com/hashicorp/vault/sdk`.
+`github.com/lf-edge/openbao/api` and `github.com/lf-edge/openbao/sdk`.
```
Comment (naphelps) [inline README.md:127]: openbao/openbao

File: README.md [hunk 5]
```diff
@@ -180,7 +180,7 @@ An example of how to use it:
 ```go
 import (
   "testing"
-  "github.com/hashicorp/vault/sdk/helper/testcluster/docker"
+  "github.com/lf-edge/openbao/sdk/helper/testcluster/docker"
```
Comment (naphelps) [inline README.md:183]: openbao/openbao

File: README.md [hunk 6]
```diff
@@ -204,7 +204,7 @@ Or for Enterprise:
...
```
Comment (naphelps) [inline README.md:207]: openbao/openbao

File: README.md [hunk 7]
```diff
@@ -237,7 +237,7 @@ func Test_Custom_Build_With_Docker(t *testing.T) {
 }
 ```
 
-There are a variety of helpers in the `github.com/hashicorp/vault/sdk/helper/testcluster`
+There are a variety of helpers in the `github.com/lf-edge/openbao/sdk/helper/testcluster`
```
Comment (naphelps) [inline README.md:240]: openbao/openbao

File: README.md [hunk 8]
```diff
@@ -284,5 +284,5 @@ Finally, here's an example of running an existing OSS docker test with a custom
 ```bash
 $ GOOS=linux make dev
 $ VAULT_BINARY=$(pwd)/bin/vault go test -run 'TestRaft_Configuration_Docker' ./vault/external_tests/raft/raft_binary
-ok      github.com/hashicorp/vault/vault/external_tests/raft/raft_binary        20.960s
+ok      github.com/lf-edge/openbao/vault/external_tests/raft/raft_binary        20.960s
```
Comment (naphelps) [inline README.md:287]: openbao/openbao

File: api/README.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
 Vault API
 =================
 
-This provides the `github.com/hashicorp/vault/api` package which contains code useful for interacting with a Vault server.
+This provides the `github.com/lf-edge/openbao/api` package which contains code useful for interacting with a Vault server.
```
Comment (naphelps) [inline api/README.md:4]: openbao/openbao

File: api/auth/approle/go.mod [hunk 10]
```diff
@@ -1,5 +1,5 @@
-module github.com/hashicorp/vault/api/auth/approle
+module github.com/lf-edge/openbao/api/auth/approle
 
 go 1.16
 
-require github.com/hashicorp/vault/api v1.9.2
+require github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/approle/go.mod:5]: openbao/openbao

File: api/auth/approle/go.sum [hunk 11]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
 github.com/hashicorp/go-sockaddr v1.0.2/go.mod h1:rB4wwRAUzs07qva3c5SdrY/NEtAUjGlgmH/UkBUC97A=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/approle/go.sum:42]: openbao/openbao

File: api/auth/approle/go.sum [hunk 12]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/approle/go.sum:43]: openbao/openbao

File: api/auth/aws/go.mod [hunk 13]
```diff
@@ -1,4 +1,4 @@
-module github.com/hashicorp/vault/api/auth/aws
+module github.com/lf-edge/openbao/api/auth/aws
```
Comment (naphelps) [inline api/auth/aws/go.mod:1]: openbao/openbao

File: api/auth/aws/go.mod [hunk 14]
```diff
@@ -7,5 +7,5 @@ require (
 	github.com/hashicorp/go-hclog v0.16.2
 	github.com/hashicorp/go-secure-stdlib/awsutil v0.1.6
 	github.com/hashicorp/go-uuid v1.0.2
-	github.com/hashicorp/vault/api v1.9.2
+	github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/aws/go.mod:10]: openbao/openbao

File: api/auth/aws/go.sum [hunk 15]
```diff
@@ -47,8 +47,8 @@ github.com/hashicorp/go-uuid v1.0.2 h1:cfejS+Tpcp13yd5nYHWDI6qVCny6wyX2Mt5SGur2I
 github.com/hashicorp/go-uuid v1.0.2/go.mod h1:6SBZvOh/SIDV7/2o3Jml5SYk/TvGqwFJ/bN7x4byOro=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/aws/go.sum:50]: openbao/openbao

File: api/auth/aws/go.sum [hunk 16]
```diff
@@ -47,8 +47,8 @@ github.com/hashicorp/go-uuid v1.0.2 h1:cfejS+Tpcp13yd5nYHWDI6qVCny6wyX2Mt5SGur2I
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/aws/go.sum:51]: openbao/openbao

File: api/auth/azure/go.mod [hunk 17]
```diff
@@ -1,5 +1,5 @@
-module github.com/hashicorp/vault/api/auth/azure
+module github.com/lf-edge/openbao/api/auth/azure
```
Comment (naphelps) [inline api/auth/azure/go.mod:1]: openbao/openbao

File: api/auth/azure/go.mod [hunk 18]
```diff
@@ -1,5 +1,5 @@
...
 
 go 1.16
 
-require github.com/hashicorp/vault/api v1.9.2
+require github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/azure/go.mod:5]: openbao/openbao

File: api/auth/azure/go.sum [hunk 19]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
 github.com/hashicorp/go-sockaddr v1.0.2/go.mod h1:rB4wwRAUzs07qva3c5SdrY/NEtAUjGlgmH/UkBUC97A=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/azure/go.sum:42]: openbao/openbao

File: api/auth/azure/go.sum [hunk 20]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/azure/go.sum:43]: openbao/openbao

File: api/auth/gcp/go.mod [hunk 21]
```diff
@@ -1,10 +1,10 @@
-module github.com/hashicorp/vault/api/auth/gcp
+module github.com/lf-edge/openbao/api/auth/gcp
 
 go 1.16
 
 require (
 	cloud.google.com/go v0.97.0
-	github.com/hashicorp/vault/api v1.9.2
+	github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/gcp/go.mod:7]: openbao/openbao

File: api/auth/gcp/go.sum [hunk 22]
```diff
@@ -187,8 +187,8 @@ github.com/hashicorp/golang-lru v0.5.0/go.mod h1:/m3WP610KZHVQ1SGc6re/UDhFvYD7pJ
 github.com/hashicorp/golang-lru v0.5.1/go.mod h1:/m3WP610KZHVQ1SGc6re/UDhFvYD7pJ4Ao+sR/qLZy8=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/gcp/go.sum:190]: openbao/openbao

File: api/auth/gcp/go.sum [hunk 23]
```diff
@@ -187,8 +187,8 @@ github.com/hashicorp/golang-lru v0.5.0/go.mod h1:/m3WP610KZHVQ1SGc6re/UDhFvYD7pJ
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/gcp/go.sum:191]: openbao/openbao

File: api/auth/kubernetes/go.mod [hunk 24]
```diff
@@ -1,5 +1,5 @@
-module github.com/hashicorp/vault/api/auth/kubernetes
+module github.com/lf-edge/openbao/api/auth/kubernetes
```
Comment (naphelps) [inline api/auth/kubernetes/go.mod:1]: openbao/openbao

File: api/auth/kubernetes/go.mod [hunk 25]
```diff
@@ -1,5 +1,5 @@
...
 
 go 1.16
 
-require github.com/hashicorp/vault/api v1.9.2
+require github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/kubernetes/go.mod:5]: openbao/openbao

File: api/auth/kubernetes/go.sum [hunk 26]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
 github.com/hashicorp/go-sockaddr v1.0.2/go.mod h1:rB4wwRAUzs07qva3c5SdrY/NEtAUjGlgmH/UkBUC97A=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/kubernetes/go.sum:42]: openbao/openbao

File: api/auth/kubernetes/go.sum [hunk 27]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/kubernetes/go.sum:43]: openbao/openbao

File: api/auth/ldap/go.mod [hunk 28]
```diff
@@ -1,5 +1,5 @@
-module github.com/hashicorp/vault/api/auth/ldap
+module github.com/lf-edge/openbao/api/auth/ldap
```
Comment (naphelps) [inline api/auth/ldap/go.mod:1]: openbao/openbao

File: api/auth/ldap/go.mod [hunk 29]
```diff
@@ -1,5 +1,5 @@
...
 
 go 1.16
 
-require github.com/hashicorp/vault/api v1.9.2
+require github.com/lf-edge/openbao/api v1.9.2
```
Comment (naphelps) [inline api/auth/ldap/go.mod:5]: openbao/openbao

File: api/auth/userpass/go.mod [hunk 30]
```diff
@@ -1,5 +1,5 @@
-module github.com/hashicorp/vault/api/auth/userpass
+module github.com/lf-edge/openbao/api/auth/userpass
```
Comment (naphelps) [inline api/auth/userpass/go.mod:1]: openbao/openbao

File: api/auth/userpass/go.sum [hunk 31]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
 github.com/hashicorp/go-sockaddr v1.0.2/go.mod h1:rB4wwRAUzs07qva3c5SdrY/NEtAUjGlgmH/UkBUC97A=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/userpass/go.sum:42]: openbao/openbao

File: api/auth/ldap/go.sum [hunk 32]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
 github.com/hashicorp/go-sockaddr v1.0.2/go.mod h1:rB4wwRAUzs07qva3c5SdrY/NEtAUjGlgmH/UkBUC97A=
 github.com/hashicorp/hcl v1.0.0 h1:0Anlzjpi4vEasTeNFn2mLJgTSwt0+6sfsiTG8qcWGx4=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
-github.com/hashicorp/vault/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
-github.com/hashicorp/vault/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
+github.com/lf-edge/openbao/api v1.9.2 h1:YjkZLJ7K3inKgMZ0wzCU9OHqc+UqMQyXsPXnf3Cl2as=
```
Comment (naphelps) [inline api/auth/ldap/go.sum:42]: openbao/openbao

File: api/auth/ldap/go.sum [hunk 33]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/ldap/go.sum:43]: openbao/openbao

File: api/auth/userpass/go.sum [hunk 34]
```diff
@@ -39,8 +39,8 @@ github.com/hashicorp/go-sockaddr v1.0.2 h1:ztczhD1jLxIRjVejw8gFomI1BQZOe2WoVOu0S
...
+github.com/lf-edge/openbao/api v1.9.2/go.mod h1:jo5Y/ET+hNyz+JnKDt8XLAdKs+AM0G5W0Vp1IrFI8N8=
```
Comment (naphelps) [inline api/auth/userpass/go.sum:43]: openbao/openbao

File: api/README.md [hunk 35]
```diff
@@ -1,9 +1,9 @@
...
 
 For examples of how to use this module, see the [vault-examples](https://github.com/hashicorp/vault-examples) repo.
 For a step-by-step walkthrough on using these client libraries, see the [developer quickstart](https://www.vaultproject.io/docs/get-started/developer-qs).
 
-[![GoDoc](https://godoc.org/github.com/hashicorp/vault/api?status.png)](https://godoc.org/github.com/hashicorp/vault/api)
\ No newline at end of file
+[![GoDoc](https://godoc.org/github.com/lf-edge/openbao/api?status.png)](https://godoc.org/github.com/lf-edge/openbao/api)
```
Comment (naphelps) [inline api/README.md:9]: openbao/openbao

File: changelog/note.tmpl [hunk 36]
```diff
@@ -1,3 +1,3 @@
 {{- define "note" -}}
-{{.Body}}{{if not (stringHasPrefix .Issue "_")}} [[GH-{{- .Issue -}}](https://github.com/hashicorp/vault/pull/{{- .Issue -}})]{{end}}
+{{.Body}}{{if not (stringHasPrefix .Issue "_")}} [[GH-{{- .Issue -}}](https://github.com/lf-edge/openbao/pull/{{- .Issue -}})]{{end}}
```
Comment (naphelps) [inline changelog/note.tmpl:2]: openbao/openbao

**Review by naphelps — CHANGES_REQUESTED** [review]
Missed some of the lf-edge repo renames to openbao/openbao. 43 occurrences.

**Review by naphelps — APPROVED** [review]

**Pull request merged**